### PR TITLE
Store configurations for client and server apps separately

### PR DESCRIPTION
**Pull request by koistya**
This should prevent accidentally leaking confidential configuration settings, a fail-safe approach.

##### `config/config.client.js`
```js
export const appName = 'React Starter Kit';
```

##### `config/config.server.js`
```js
export const databaseUrl =
  process.env.DATABASE_URL || 'postgresql://postgres@localhost:5433/database';

export const facebook = {
  id: process.env.FACEBOOK_ID || '<your-facebook-id>',
  secret: process.env.FACEBOOK_SECRET || '<your-facebook-secret>',
};

export { appName } from './config.client.js';
```

##### `config/package.json`
```json
{
  "name": "config",
  "main": "./config.server.js",
  "browser": "./config.client.js"
}
```

Note: The same approach is currently used in the HTTP client utility (`src/core/fetch`).

### How to Use

```js
import { appName, databaseUrl } from '../config';

console.log(appName); -> available in both client-side and server-side code
console.log(databaseUrl); -> available only in server-side code
```

**Pro Tip**: If you're using PostgreSQL, you can set password for the `postgres` user from the connection string above via [.pgpass](http://www.postgresql.org/docs/current/static/libpq-pgpass.html) file, so that the same connection string above could work for all team members. Also, all hosting providers (e.g. Heroku, OpenShift, Azure, Bluemix) allow you to configure environment variables, no need to store any actual passwords in your GitHub repository.

For review @frenzzy, @Bogdaan, all.

### CODE REVIEW

**Comment by Bogdaan** [comment]
:white_check_mark: good approach )

**Comment by koistya** [comment]
Mm.. over-engineering! As it is now is good enough.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ visit our sponsors:
 │   ├── /actions/               # Action creators that allow to trigger a dispatch to stores
 │   ├── /api/                   # REST API / Relay endpoints
 │   ├── /components/            # React components
+│   ├── /config/                # Configuration options for both client and server apps
```
Comment (frenzzy) [inline README.md]: Did you forget to update this comment?

File: src/config/package.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "name": "config",
+  "main": "./config.server.js",
+  "browser": "./config.client.js"
```
Comment (frenzzy) [inline src/config/package.json]: It would be nice to remove client config if no longer needed

**Comment by koistya** [comment]
Closing in favor of 6878c79dd6974e6d4571601b3a00863b3a850211

**Pull request closed without merging**